recipes/dfast/meta.yaml

```diff
@@ -3,29 +3,24 @@ package:
   version: 1.2.3
 
 build:
-  number: 0
-  noarch: generic
+  number: 1
+  noarch: python
```

Comment:
It doesn't really matter, but this is in fact `noarch: generic` (`noarch: python` is only needed for traditional python packages (i.e., things installed with `pip install`)).

Reply:
Actually, this should have never been a generic package, since it's vendoring `last` and probably some other packages :(

Reply:
@dpryan79 Thank you for the comments.
Actually, I just wanted to add BLAST dependency ">=2.6.0", but build test fails for some reason despite that it succeeded last month.
I'm going to try again.